### PR TITLE
fix issue #4109 related with Fira fonts on the button and placeholders

### PR DESCRIPTION
**Pull request by krydos**
Hello. I hope I fixed this problem. Could you verify this pull request please and merge it if you not mind.

### CODE REVIEW

**Comment by jaredhirsch** [comment]
Hey @KryDos, thanks for submitting this PR!

We're currently focused on getting a release out the door, but I'll take a look once that's done.

**Comment by shane-tomlinson** [comment]
Hey @KryDos! Good to see you here!

File: resources/static/dialog/css/m.css [hunk 1]
```diff
@@ -27,6 +27,13 @@
   body {
     height: 100%;
     background: #eff0f2 none;
+  }
+
+  /**
+   * Elements like buttons, selects and inputs don't inherit font styles from body
```
Comment (shane-tomlinson) [inline resources/static/dialog/css/m.css:33]: You know, I just learned this about a month ago. :/

**Comment by krydos** [comment]
@6a68 Thank you.
@shane-tomlinson I'm glad that you remember me :)

> You know, I just learned this about a month ago. :/

I learned it when I was working on this bug ;) Very strange behavior but I'm sure that browser developers had serious reasons for this.

File: resources/static/dialog/css/m.css [hunk 2]
```diff
@@ -27,6 +27,13 @@
...
+   * so we have to set the font styles for each type of this elements.
+   */
+  body, button, input {
     font-family: 'Fira Sans', 'Feura Sans', 'Open Sans', 'Lucida Sans', 'Lucida Grande', 'Lucida Sans Unicode', Verdana, sans-serif;
```
Comment (jaredhirsch) [inline resources/static/dialog/css/m.css:37]: is there a reason we can't just set `font-family: inherit`?

**Comment by krydos** [comment]
@6a68, I'm sorry but I didn't know about **inherit** option for **font-family** attribute. At the moment I don't know how does it work. But I will check it. Thank you.

**Comment by jaredhirsch** [comment]
@KryDos Awesome! See if it works, if it does, that will let us avoid duplicating the font-family string here.

**Comment by shane-tomlinson** [comment]
@KryDos - nor did I. It's listed in https://developer.mozilla.org/en-US/docs/Web/CSS/font-family?redirectlocale=en-US&redirectslug=CSS%2Ffont-family as valid.

**Comment by krydos** [comment]
Please excuse me for the delay. I will check this issue on the weekend. Thank you.

**Comment by krydos** [comment]
Hi guys. I checked the "inherit" option for the "font-family" and it doesn't work for me (Firefox 30) :(

**Comment by krydos** [comment]
A little bit more information:
if I enables the "inherit" option then fonts inherit something but not that we need (we need Fira Sans).

**Comment by callahad** [comment]
Merged in https://github.com/mozilla/persona/commit/677bb87fae5a587889f17461eaaaf7d3c76dad0a

Thanks, @KryDos !

**Pull request closed without merging**